notebooks/non-sequential.ipynb

```diff
@@ -0,0 +1,453 @@
+{
```

Comment:
<p>I am a bit torn on whether or not to include the pywatts material, so will just list some thoughts I have on the matter:</p><p> </p><ul><li>I think anytime you include actual code you sort of invite people to copy/use it themselves.  Which given that the code is still experimental, may not be the best idea.</li><li>If we are still thinking about changing the API or interface at all (which I think was the intention - yes?) then I hesitate to give concrete code at this stage. </li><li>Personally I am not opposed to the fact that the code doesn't live in sktime, I think we can clarify that it is the result of a collaboration, so this point is ok to me. </li><li>I think the advantage of including the code it is makes it easier to see how one might actually be able to create a graphical pipeline, which may not be obvious from the images/abstract discussion. </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime-tutorial-pydata-global-2022/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
I removed it from the notebook.